src/Standards/PSR12/Docs/ControlStructures/ControlStructureSpacingStandard.xml

```diff
@@ -0,0 +1,124 @@
+<documentation title="Control Structure Spacing">
+    <standard>
+    <![CDATA[
+    The control structures must have one space after the control structure keyword, no spaces after the opening parenthesis and before the closing parenthesis, and one space between the closing parenthesis and the opening brace.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: No space after the opening parenthesis in a single-line condition.">
+        <![CDATA[
+if <em>(</em>$expr) {
+}
+        ]]>
+        </code>
+        <code title="Invalid: Space after the opening parenthesis in a single-line condition.">
+        <![CDATA[
+if <em>( </em>$expr) {
+}
+        ]]>
+        </code>
+    </code_comparison>
+    <code_comparison>
+        <code title="Valid: No space before the closing parenthesis in a single-line condition.">
+        <![CDATA[
+if <em>(</em>$expr) {
+}
+        ]]>
+        </code>
+        <code title="Invalid: Space before the closing parenthesis in a single-line condition.">
+        <![CDATA[
+if ($expr<em> </em>) {
+}
+        ]]>
+        </code>
+    </code_comparison>
+    <standard>
+    <![CDATA[
+    The body of the multi-line control structure must be indented once, placing the body on the next line after the opening brace.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: First expression of a multi-line control structure condition block is on the line after the opening parenthesis.">
+        <![CDATA[
+while (
+    <em>$expr1</em>
+    && $expr2
+) {
+}
+        ]]>
+        </code>
+        <code title="Invalid: First expression of a multi-line control structure condition block is on the same line as the opening parenthesis.">
+        <![CDATA[
+while (<em></em>$expr1
+    && $expr2
+) {
+}
+        ]]>
+        </code>
+    </code_comparison>
+    <code_comparison>
+        <code title="Valid: Each line in a multi-line control structure condition block indented at least once. Default indentation is 4 spaces.">
+        <![CDATA[
+while (
+<em>    </em>$expr1
+<em>    </em>&& $expr2
+) {
+}
+        ]]>
+        </code>
+        <code title="Invalid: Some lines in a multi-line control structure condition block not indented correctly.">
+        <![CDATA[
+while (
+<em></em>$expr1
+    && $expr2
+<em>  </em>&& $expr3
+) {
+}
+        ]]>
+        </code>
+    </code_comparison>
+    <standard>
+    <![CDATA[
+    The closing parenthesis of the multi-line control structure must be on the next line after the body, indented to the same level as the start of the control structure.
```

Comment:
Dito. This is not about the control structure body, but about multi-line conditions.